main/src/addins/VersionControl/MonoDevelop.VersionControl/MonoDevelop.VersionControl/UrlBasedRepository.cs

```diff
@@ -79,7 +79,7 @@ public virtual bool IsUrlValid (string url)
 		public virtual string Url
 		{
 			get { return url; }
-			set { url = value; CreateUri (); }
+			set { url = SanitizerUrl (value); CreateUri (); }
```

Comment:
Potential NRE (CreateUri () has a null check), let's add a null check to `SanitizerUrl ()`, or – even better – just simplify this to:
```suggestion
			set { url = value?.Trim (); CreateUri (); }
```

Reply:
Changed!